tools/e2e/common.ts

```diff
@@ -288,3 +290,70 @@ export const listCertificates = async () => {
 export const deleteCertificate = async (id: string) => {
 	await apiFetch(`/mtls_certificates/${id}`, "DELETE");
 };
+
+// Note: the container images functions below don't directly use the REST API since
+//       they interact with the cloudflare images registry which has it's own
+//       non-trivial auth mechanism, so instead of duplicating a bunch of logic
+//       here we instead directly call wrangler
+//
+// TODO: Consider if it would make sense for all the functions here to use wrangler
+//       directly, what is the advantage of hitting the REST API directly?
+
+export const listE2eContainerImages = () => {
+	const output = runWranglerCommand("wrangler containers images list", {
+		// This operation can take literally minutes to run...
+		timeout: 3 * 60_000,
```

Comment:
this seemed necessary for me locally... but maybe it not a general issue... 🤔 🤷 

I can try to set this to something more reasonable and see if it still works in CI? 🤔

(although a big timeout here probably doesn't have huge downsides 🤔)

WDYT @emily-shen?